src/components/ifxTabs/ifxTabs.stories.tsx

```diff
@@ -0,0 +1,46 @@
+export default {
+  title: 'Components/Tabs',
+  component: 'ifx-tabs',
+  argTypes: {
+    amountOfTabs: { control: 'number' },
```

Comment:
When `amountOfTabs` is `0`, there is an error in the console. 
Why not rename `amountOfTabs` to `tabs`. The word 'amount' is not correct here, and there should be space between the words.

Reply:
Fixed the console.log error
Added names. We should propably add this to every story.